examples/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test-unit": "pnpm test __tests__/unit/",
     "test-e2e": "pnpm test __tests__/integration/",
-    "test": "jest --env node --verbose --runInBand --bail"
+    "test": "jest --reporters=default --reporters=jest-junit --env node --verbose --runInBand --bail"
```

Comment:
Can we move these to the jest config file instead of command line flags?

Reply:
This was what I [tried initially](https://github.com/vercel/vercel/pull/10334/commits/c9a6dfec3328e675a9a0ed256d2766367ec72666#diff-ac4313523217deffcd0ebf639a98c806b787a42b7d6440853ce9149a06a43cc5) but these files weren't being sourced in CI. Probably why all the args are inlined.